### PR TITLE
fixes / additions to README.md

### PR DESCRIPTION
**Pull request by cmcghan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,9 +61,37 @@ sudo apt-get install git
...
+webgme start
+```
+
+Finally, navigate to `http://localhost:8888` to start using the WebGME-BIP design studio!
+```
+firefox "https://127.0.0.1:8888/" &
+```
+
+As an aside... you can also download and install `webgme` itself locally if you'd like to play around with the basic (non-bip) interface:
+```
```
Comment (pmeijer) [inline README.md]: You can do this from the bip repository. It contains webgme as a node_module and all "generic" visualization is included.

File: README.md [hunk 2]
```diff
@@ -55,9 +61,37 @@ sudo apt-get install git
...
+mkdir bip_data
+mongodb --dbpath ./bip_data
+```
+wait until you see a line that says "[initandlisten] waiting for connections on port 27017".
 
-Then, run `webgme start` from the project root to start . Finally, navigate to `http://localhost:8888` to start using the WebGME-BIP design studio!
+Then, in a new terminal window, run `webgme start` from the project root (`webgme-bip`) to start. For example:
+```
+cd /home/$USER/webgme-bip
+webgme start
```
Comment (pmeijer) [inline README.md]: Once you've done `npm install` `npm start` suffices here.

**Comment by cmcghan** [comment]
I haven't tried all combinations on a fresh Linux base, and installing webgme-bip was my first introduction to npm. So, if you have suggestions for additions/changes/etc. to my additions to README.md, then please add them! I wasm't expecting these to necessary be in the best format (I assumed they'd require further edits and refinements by the professionals ;) -- I was mainly just trying to share what I messed around with that got webgme-bip working for me (under Ubuntu 16.04 LTS). Better instructions would be a win in my book :) Sorry for not making this clear in my pull request comment above.

**Comment by pmeijer** [comment]
Sorry if I came across as arrogant. Really appreciate the PR. My comments are minor technicalities really. @anmavrid is the owner of the repo and I just wanted to clear out a couple of things before the merge.

**Pull request merged**